### ISSUE TITLE
IANSCU queue  have error in random studies

### ISSUE THREAD

**Issue by lsoeot** [dcm4che/dcm4chee-arc-light#1570]
hello, 
i've notice after runing IAN service on queue are many ,,SCHEDULED" rows... when i've check it i've seen it's because last  transmit was with error "Unknown AE:" and it's true there is no remote AET in this details but why, many studies work correct ... it's total random error is like in screenshot below

![image](https://user-images.githubusercontent.com/38498348/45100903-a2085d00-b12a-11e8-8b81-47a4c6518c13.png)

**Comment by gunterze** [comment]
Check your configured IAN Forward Destinations. The failed will be rescheduled according the configured retry parameters of the IANSCU queue.

**Comment by lsoeot** [comment]
yes i get it , but why there is fail in first place 
destination's was ok ...

![image](https://user-images.githubusercontent.com/38498348/45103938-4392ad00-b131-11e8-890c-0881f979c0f9.png)


this random'ness is what bother me

![image](https://user-images.githubusercontent.com/38498348/45104103-8fdded00-b131-11e8-9b22-5a2bf8837542.png)

**Comment by gunterze** [comment]
The UI may hide invalid IAN forward destinations. Check directly in LDAP.

**Comment by lsoeot** [comment]
you are right in ldap there is some empty destination not visible in UI ! thx (but i think it should be bug because lack of this in UI ) 
![image](https://user-images.githubusercontent.com/38498348/45147467-43dc8800-b1c5-11e8-8a6a-403421e24d89.png)
